stable/fairwinds-insights/templates/deployment-open-api.yaml

```diff
@@ -35,6 +35,14 @@ spec:
             value: {{ "https://" }}{{ include "fairwinds-insights.sanitizedPrefix" $ }}{{ index .Values.ingress.hostedZones 0 }}{{ "/v0/swagger.yaml" }}
           - name: BASE_URL
             value: "/swagger"
+          livenessProbe:
+            httpGet:
+              path: /swagger
+              port: http
+          readinessProbe:
+            httpGet:
+              path: /swagger
+              port: http
```

Comment:
Where is the current health probe coming from? Something is trying to access `/health`

Reply:
I think if you don't specify health probes, datadog daemonset assumes the default health probes to be `/health for some reason.

Reply:
well that's weird! Thanks for the explanation